src/form_builder/models.py

```diff
@@ -51,10 +51,12 @@ class Form(models.Model):
                              max_length=255,
                              help_text=_("Give your form a name."))
     slug = models.SlugField(_("Slug"),
-                            editable=False,
+                            editable=True,
                             unique=True,
                             max_length=255,
-                            blank=True)
+                            blank=True,
+                            help_text=_("This will modify the URL.")
```

Comment:
:+1: to Jimmy's suggestion, but I think this should be pretty verbose to explain what the end result of the field will be. Something like:

"Choose a custom URL slug for this form. For example, if you enter `my-first-form`, the URL to share with respondent will be `{{ SOME_BASE_URL_VAR_I_CAN'T_REMEMBER_THE_NAME_OF }}`/forms/respond/**my-first-form**/. If not set, the default is to use the form's name in lowercase with dashes instead of spaces."